lib/ruby_indexer/lib/ruby_indexer/entry.rb

```diff
@@ -498,12 +501,13 @@ def initialize(target, unresolved_alias)
         )
 
         @target = target
+        @owner = T.let(unresolved_alias.owner, T.nilable(Entry::Namespace))
       end
 
-      sig { returns(T.nilable(Entry::Namespace)) }
-      def owner
-        @target.owner
-      end
+      # sig { returns(T.nilable(Entry::Namespace)) }
```

Comment:
Was this change intentional?

Reply:
Ah, forgot to remove the commented out code. I removed it directly on main https://github.com/Shopify/ruby-lsp/commit/3aa04306728a7cf24a1a1b9432154ea9f1ae3260.